### DISCUSSION TITLE
Trying to get this simple config to work

### DISCUSSION THREAD

**Discussion by rijckholt** [maca88/SmartBikeLights#36] [Q&A] [answered]
Hi All,

A a first simple setup I am trying to achieve the following with Garmin edge1040 and RTL510:

Default state of the RTL50 is OFF as i have other (fixed) lights on my bike and only want it to FLASH if cars approach....

that's it...... looks simple enough but i can't get this to work. I tired some the other configs i saw here but nothing seems to work. So so wondering: what am I missing here.....

My config is as follows:

###0,73404416#2,2!Flash:1:7:1:0I[140]1!:1:0:0:0D=1##5,3:RTL510!2,:-1,Off:0!1,High:4!2,Day Flash:7,Night Flash:6#0::#0:0#123!:123!#B3843##2#0#0

**Comment by maca88** [comment, marked as answer]
Hi,

With your configuration the light will start flash only when a very fast vehicle is approaching you. Try to use the following configuration that will turn on your Varia on every approaching vehicle:
###0,73404416#2,2!Flash:1:7:0:0I]-1]0!:1:0:0:0D=1##5,3:RTL510!2,:-1,Off:0!1,High:4!2,Day Flash:7,Night Flash:6#0::#0:0#123!:123!#B3843##2#0#0

I removed the range filter as Varia does not detect vehicles further than 140 meters based on the specification. Also, make sure that you set the light in Smart mode (S should be displayed on the top left button/light icon)

**Reply by rijckholt** [reply]
Hi Maca88,

Thanks so much for your quick reply. Just tested this and it works! Awesome ;-) 

It also immediately sparks the next question: What would be the best approach to configure a profile for my other bike without fixed lights

So 
- Bike1, with fixed lights only flash when cars approach (that is config from above)
- Bike2, without fixed lights during sunrise only flash when cars approach, sunset use flash mode (i think that mode already changes the flash rhythm when cars approach) 

Would the following config using different activity profiles achieve that? 

###0,73404416#6,3!Bike1M:2:7:0:0K=BIKE1:I]-1]0!Bike2M:3:7:0:0K=BIKE2:Es0,r0I]-1]0!:1:0:0:0D=1##5,3:RTL510!2,:-1,Off:0!1,High:4!2,Day Flash:7,Night Flash:6#0::#0:0#123!:123!#B3843##2#0#0

Thanks in advance for your time and effort.

**Comment by maca88** [comment]
Hi,

you can use the following configuration:
###0,73404416#4,3!Bike2M:2:6:0:0K=BIKE2:Es0,r0!RADAR:1:7:0:0I]-1]0!:1:0:0:0D=1##5,3:RTL510!2,:-1,Off:0!1,High:4!2,Day Flash:7,Night Flash:6#0::#0:0#123!:123!#B3843##2#0#0

RADAR filter is used for both bikes for approaching vehicles and Bike2M will be activated only for Bike2 after sunset with Night Flash mode.
